app/controllers/auth/AuthAction.scala

```diff
@@ -28,13 +28,17 @@ import uk.gov.hmrc.domain._
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.HeaderCarrierConverter
 import uk.gov.hmrc.play.bootstrap.auth.DefaultAuthConnector
+
 import scala.concurrent.{ExecutionContext, Future}
 
-class AuthActionImpl @Inject()(override val authConnector: DefaultAuthConnector, configuration: Configuration)(
-  implicit ec: ExecutionContext,
-  implicit val appConfig: ApplicationConfig)
+//TODO look at the amount of config being passed
```

Comment:
Can you explain this comment? Do you need a ticket for this?

Reply:
needed to check whether we need to pass the configuration or not, turns out we dont hence removed the configuration as part of code clean up